WeakAuras/Init.lua

```diff
@@ -518,12 +517,6 @@ do
         GameTooltip:Hide()
       end,
     })
-    if C_AddOns.IsAddOnLoaded("!!WWAddOnsFix") then
-      noIncompatibleAddon = false
-      C_Timer.After(1, function()
-        WeakAuras.prettyPrint("Incompatible addon detected: please disable '!!WWAddOnsFix'")
```

Comment:
Wasn't there a problem with the compability functions !!WWAddOnsFix provided, where they only returned partial information?